### PR TITLE
Update 'oc rsync' documentation

### PR DESCRIPTION
**Pull request by csrwng**
Updates to match functionality in origin. The copy may happen from
container to pod or from pod to container. Rsync is also now supported
in Windows.

### CODE REVIEW

**Comment by csrwng** [comment]
@bparees @adellape ptal

File: dev_guide/copy_files_to_container.adoc [hunk 1]
```diff
@@ -26,28 +26,47 @@ For example:
...
 The `oc rsync` command uses the local `rsync` command if present on the client's machine. This requires
-that the remote container also have the `rsync` command.  If `rsync` is not found locally or in the remote 
+that the remote container also have the `rsync` command.  If `rsync` is not found locally or in the remote
 container, then a tar archive will be created locally and sent to the container where `tar` will be used to
 extract the files. If `tar` is not available in the remote container, then the copy will fail.
 
 [NOTE]
 ====
-On Windows clients, `oc rsync` currently only supports tar.
+The tar copy method does not provide the same functionality as rsync. For example, rsync will create a
```
Comment (bparees) [inline dev_guide/copy_files_to_container.adoc]: s/create a/create the/

**Comment by bparees** [comment]
one nit and lgtm.

**Comment by csrwng** [comment]
updated

**Comment by adellape** [comment]
:+1:

**Comment by vikram-redhat** [comment]
Doesn't require a followup.

**Pull request merged**